quic/s2n-quic/src/tests/snapshots/tests__pto__handshake_pto_timer_is_armed__events.snap

```diff
@@ -26,7 +26,6 @@ count#mtu_updated=1
 measure#mtu_updated.mtu=1200b
 count#mtu_updated.cause|NEW_PATH=1
 count#mtu_updated.search_complete=false
-count#server_name_information=1
```

Comment:
Note that here the Client ends up not emitting the server name as in this test the server never sends any packets to the client.